File: generate/server/java/src/main/java/io/swagger/server/api/model/Todolist.java
package io.swagger.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import java.math.BigDecimal;
import java.util.ArrayList;
import java.util.List;
import io.vertx.core.json.JsonObject;
import io.vertx.codegen.annotations.DataObject;
/**
 * Todolist
 */

@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaVertXServerCodegen", date = "2023-10-11T18:50:12.795023015Z[GMT]")



public class Todolist   {
  private String id = null;

  private String name = null;

  private BigDecimal priority = null;

  
  private List<String> tags = null;

  public Todolist id(String id) {
    this.id = id;
    return this;
  }

  /**
   * Get id
   * @return id
   **/
    public String getId() {
    return id;
  }

  public void setId(String id) {
    this.id = id;
  }

  public Todolist name(String name) {
    this.name = name;
    return this;
  }

  /**
   * Get name
   * @return name
   **/
    public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public Todolist priority(BigDecimal priority) {
    this.priority = priority;
    return this;
  }

  /**
   * Get priority
   * @return priority
   **/
    public BigDecimal getPriority() {
    return priority;
  }

  public void setPriority(BigDecimal priority) {
    this.priority = priority;
  }

  public Todolist tags(List<String> tags) {
    this.tags = tags;
    return this;
  }

  public Todolist addTagsItem(String tagsItem) {
    if (this.tags == null) {
      this.tags = new ArrayList<String>();
    }
    this.tags.add(tagsItem);
    return this;
  }

  /**
   * Get tags
   * @return tags
   **/
    public List<String> getTags() {
    return tags;
  }

  public void setTags(List<String> tags) {
    this.tags = tags;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    Todolist todolist = (Todolist) o;
    return Objects.equals(this.id, todolist.id) &&
        Objects.equals(this.name, todolist.name) &&
        Objects.equals(this.priority, todolist.priority) &&
        Objects.equals(this.tags, todolist.tags);
  }

  @Override
  public int hashCode() {
    return Objects.hash(id, name, priority, tags);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class Todolist {\n");
    
    sb.append("    id: ").append(toIndentedString(id)).append("\n");
    sb.append("    name: ").append(toIndentedString(name)).append("\n");
    sb.append("    priority: ").append(toIndentedString(priority)).append("\n");
    sb.append("    tags: ").append(toIndentedString(tags)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: generate/client/dart/lib/model/todolist.dart
part of swagger.api;

class Todolist {
  
  String id = null;

  String name = null;

  double priority = null;

  List<String> tags = [];

  Todolist();

  @override
  String toString() {
    return 'Todolist[id=$id, name=$name, priority=$priority, tags=$tags, ]';
  }

  Todolist.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    id = json['id'];
    name = json['name'];
    priority = json['priority'];
    tags = (json['tags'] as List).map((item) => item as String).toList();
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'name': name,
      'priority': priority,
      'tags': tags
     };
  }

  static List<Todolist> listFromJson(List<dynamic> json) {
    return json == null ? new List<Todolist>() : json.map((value) => new Todolist.fromJson(value)).toList();
  }

  static Map<String, Todolist> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, Todolist>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new Todolist.fromJson(value));
    }
    return map;
  }
}
